src/main/java/dev/openfeature/javasdk/Value.java

```diff
@@ -51,38 +55,38 @@ public Value(ZonedDateTime value) {
     }
 
     /** 
-     * Check if this Value represents a Boolean.
+     * Check if this Value represents null.
      * 
      * @return boolean
      */
-    public boolean isBoolean() {
-        return this.innerObject instanceof Boolean;
+    public boolean isNull() {
+        return this.innerObject == null;
     }
 
     /** 
-     * Check if this Value represents a String.
+     * Check if this Value represents a Boolean.
      * 
      * @return boolean
      */
-    public boolean isString() {
-        return this.innerObject instanceof String;
+    public boolean isBoolean() {
+        return this.innerObject instanceof Boolean;
     }
 
     /** 
-     * Check if this Value represents an Integer.
+     * Check if this Value represents a String.
      * 
      * @return boolean
      */
-    public boolean isInteger() {
-        return this.innerObject instanceof Integer;
+    public boolean isString() {
+        return this.innerObject instanceof String;
     }
 
     /** 
-     * Check if this Value represents a Double.
+     * Check if this Value represents a numeric value.
      * 
      * @return boolean
      */
-    public boolean isDouble() {
+    public boolean isNumber() {
         return this.innerObject instanceof Double;
     }
```

Comment:
Unfortunately, adding the new constructor makes git see all these as deltas. :weary: